src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -472,6 +475,41 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 	 */
 	obj.getPurchaserData = () => tribe.tickets.commerce.getPurchaserData( $( tribe.tickets.commerce.selectors.purchaserFormContainer ) );
 
+	/**
+	 * Shows the notice for the checkout container for Stripe.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container Parent container of notice element.
+	 * @param {string} title Notice Title.
+	 * @param {string} content Notice message content.
+	 */
+	obj.showNotice = ( $container, title, content ) => {
+		if ( ! $container || ! $container.length ) {
+			$container = $( tribe.tickets.commerce.selectors.checkoutContainer );
+		}
+		const notice = tribe.tickets.commerce.notice;
+		const $item = $container.find( notice.selectors.item );
+		notice.populate( $item, title, content );
+		notice.show( $item );
+	};
+
+	/**
+	 * Hides the notice for the checkout container for Stripe.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container Parent container of notice element.
+	 */
+	obj.hideNotice = ( $container ) => {
+		if ( ! $container.length ) {
+			$container = $( tribe.tickets.commerce.selectors.checkoutContainer );
+		}
+
+		const notice = tribe.tickets.commerce.notice;
+		const $item = $container.find( notice.selectors.item );
+		notice.hide( $item );
```

Comment:
Let's toggle the `.tribe-common-a11y-hidden` class rather than using the `hide`. Check how we do it with the loader https://github.com/the-events-calendar/event-tickets/blob/master/src/resources/js/v2/tickets-loader.js#L53

Reply:
So, this is just using the `notice.js` component's show and hide method.
https://github.com/the-events-calendar/event-tickets/blob/db8cb39e11ab7e7f4cdb7f4c18a64cd0167c0103/src/resources/js/commerce/notice.js#L82

If we want to change the behavior within the component then it is better to do that in a separate PR as this will require to update the related CSS: https://github.com/the-events-calendar/event-tickets/blob/abf1a9b8b5e9e203fd58b3c086e2bafe68026cf4/src/resources/postcss/components/_notice.pcss#L30

Reply:
created a separate PR for the notice component update.

https://github.com/the-events-calendar/event-tickets/pull/2320